src/Fixer/PhpBasic/CodeStyle/MethodNamingFixer.php

```diff
@@ -94,36 +93,41 @@ protected function applyFix(SplFileInfo $file, Tokens $tokens): void
             $functionTokenIndex = $tokens->getPrevNonWhitespace($key);
             $visibilityTokenIndex = $functionTokenIndex ? $tokens->getPrevNonWhitespace($functionTokenIndex) : null;
 
-            if ($functionTokenIndex && $visibilityTokenIndex) {
-                if (
-                    $token->isGivenKind(T_STRING)
-                    && $tokens[$key + 1]->equals('(')
-                    && $tokens[$functionTokenIndex]->isGivenKind(T_FUNCTION)
-                    && $tokens[$visibilityTokenIndex]->isGivenKind([T_PUBLIC, T_PROTECTED, T_PRIVATE])
-                ) {
-                    $functionName = $tokens[$key]->getContent();
-                    $parenthesesEndIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $key + 1);
-                    $nextIndex = $tokens->getNextMeaningfulToken($parenthesesEndIndex);
-
-                    $returnType = null;
-                    if ($tokens[$nextIndex]->isGivenKind(CT::T_TYPE_COLON)) {
-                        $typeIndex = $tokens->getNextMeaningfulToken($nextIndex);
-                        $returnType = $tokens[$typeIndex]->getContent();
-                        $nextIndex = $tokens->getNextMeaningfulToken($typeIndex);
-                    }
-
-                    if (!$tokens[$nextIndex]->equals('{')) {
-                        continue;
-                    }
-
-                    $this->fixMethod($tokens, $functionName, $visibilityTokenIndex, $nextIndex, $returnType);
+            if (
+                $functionTokenIndex
+                && $visibilityTokenIndex
```

Comment:
strict comparison is needed, also IMO guard clause would make things easier to read, eg.
```
if ($functionTokenIndex == null || $visibilityTokenIndex === null) {
                continue;
            }
```

Reply:
Fixed. But not $functionTokenIndex == null but $functionTokenIndex === null, right ;)